app/controllers/tech_blog_controller.rb

```diff
@@ -11,11 +19,51 @@ def visits_per_technology
     Builders::BlogMetricChart::TechnologyVisits.call
   end
 
-  def visits_growth_mom
-    Builders::BlogMetricChart::TechnologyVisitsGrowthMom.call
+  def year_to_date_visits
+    Builders::ChartSummary::YearToDateBlogVisits.call.to_i
+  end
+
+  def month_to_date_visits
+    totals_for(@visits_per_technology)[:data][current_month_key].to_i
   end
 
   def technology_blog_post_count
     Builders::BlogMetricChart::TechnologyBlogPostCount.call
   end
+
+  def year_to_date_new_blog_posts
+    Builders::ChartSummary::YearToDateNewBlogPosts.call
+  end
+
+  def month_to_date_new_blog_posts
+    Builders::ChartSummary::MonthToDateNewBlogPosts.call
+  end
+
+  def visits_growth_mom
+    Builders::BlogMetricChart::TechnologyVisitsGrowthMom.call
+  end
+
+  def this_month_visits_growth
+    totals_for(@visits_growth_mom)[:data][current_month_key].round(2)
+  end
+
+  def last_month_visits_growth
+    totals_for(@visits_growth_mom)[:data][last_month_key].round(2)
+  end
+
+  def totals_for(metric_datasets)
+    metric_datasets.find { |metric_dataset| metric_dataset[:name] == 'Totals' }
+  end
```

Comment:
This method seems to belong to a BlogMetricsDataset class

It would allow to do

```ruby
@visits_growth_mom.totals[:data][last_month_key].round(2)
```

or even

```ruby
@visits_growth_mom.totals_at(last_month_key)
```

Reply:
Oh, I hadn't thought about that! Love the idea!